<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetCategoriesDialog</class>
 <widget class="QDialog" name="SetCategoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>364</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSetCategories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sector</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResRange">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rango residencial</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Residencial</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonCommercial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comercio y Servicios</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonIndustrial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Industrial</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPumping">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bombeo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPublicLight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alumbrado público</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50 kWh/mes</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>21</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50 - 100 kWh/mes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResC">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>101 - 150 kWh/mes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResD">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>151 - 250 kWh/mes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResE">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>251 - 500 kWh/mes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonResF">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;500 kWh/mes</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>21</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelSubCommercial">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subcategorías comercio</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFinances">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Financieras (bancos)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonWholesale">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comercio al por mayor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRetail">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comercio al por menor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGov">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gobierno</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonHealth">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Salud</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonHotels">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hoteles</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonEducation">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Educación</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOtherCommercial">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Otros</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>21</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelSubIndustrial">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subcategorías industria</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFood">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alimentos y bebidas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPaper">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Papel, cartón</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPlastics">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Caucho, plástico</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonChemistry">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Química</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTextiles">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Textil</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOtherIndustrial">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Otros</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectFrequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione frecuencia de las mediciones:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton15min">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>15 min</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1h">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1 h</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectDateFormat">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione formato de la fecha:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDateFormat">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>21</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
